.github/workflows/sonarcloud-analysis.yml

```
@@ -0,0 +1,39 @@
+        required: true
+      SONAR_HOST_URL:  # SonarCloud host URL
+        required: true
+
+jobs:
+  sonarcloud-analysis:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)